context_test.go

```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/golang/dep/internal/test"
 )
 
-var (
-	discardLogger = log.New(ioutil.Discard, "", 0)
-)
+func discardLogger() *log.Logger {
```

Comment:
What do you think about moving this to `internal/test` and exporting? That way it doesn't need to be defined.

Reply:
I'm kinda meh about that - tbh I was kinda meh about my original suggestion, but the idea of exporting it just makes it seem definitively unwise 😛

Reply:
I don't care much about if we do it or not but out of curiosity what's wrong/unwise with exporting `test.DiscardLogger()` in our internal test package?

Reply:
it's a really subjective thing, which is why i sorta hesitate about it in the first place, but to my mind, exporting jumps the line between an innocuous helper and a recommended pattern.